lib/UI/workout/workout_page.dart

```diff
@@ -96,11 +95,7 @@ class _WorkoutPageState extends State<WorkoutPage> {
             backgroundColor: theme.colorTheme.backgroundColorVariation,
             leading: BackButton(
               onPressed: () {
-                Navigator.of(context).pushReplacement(
-                  MaterialPageRoute(
-                    builder: (context) => CalendarScreen(),
-                  ),
-                );
+                Navigator.pop(context);
```

Comment:
You can do:

`onPressed: () => Navigator.pop(context)` or` onPressed: Navigator.of(context).pop;`